tests/contrib/test_sqlalchemy.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+import unittest
+
+from sqlalchemy import Column, create_engine, Integer
```

Comment:
Consider following the style guide's rule to [import only modules and not classes, functions, or other code elements of modules](https://google.github.io/styleguide/pyguide.html?showone=Imports#Imports).

It's true that most of the existing `oauth2client` codebase doesn't follow this rule; I apologize for that.